docs/CHANGELOG.md

```diff
@@ -3,6 +3,11 @@
 
 ## 1.0
 
+### 2.0.0
+
+- **[breaking change]** Make `:orc` table format to have `orc.column.index.access` disabled by default. `:orc_legacy` preset replaces the previous one
```

Comment:
```suggestion
- **[breaking change]** Make `:orc` table format use column names instead of indexes by default. `:orc_legacy` preset preserves existing behavior.
```